src/com/facebook/buck/android/TrimUberRDotJava.java

```diff
@@ -58,6 +58,9 @@
   private static final Pattern R_DOT_JAVA_LINE_PATTERN = Pattern.compile(
       "^ *public static final int(?:\\[\\])? (\\w+)=");
 
+  private static final Pattern R_DOT_JAVA_PACKAGE_NAME_PATTERN = Pattern.compile(
+      "^ *package(?:\\[\\])? ([\\w\\.]+);");
```

Comment:
The `?:\\[\\])?` part I took from the regex above. I dont think it is needed. I can remove it for simplicity